internal/bminventory/inventory.go

```diff
@@ -2468,6 +2471,14 @@ func (b *bareMetalInventory) setProxyUsage(httpProxy *string, httpsProxy *string
 	}
 }
 
+func (b *bareMetalInventory) setUserManagedNetworkingAndMultiNodeUsage(userManagedNetworking *bool, highAvailabilityMode *string, usages map[string]models.Usage) {
+	if userManagedNetworking == nil || highAvailabilityMode == nil {
```

Comment:
Under what circumstances is highAvailabilityMode nil?

Reply:
Yes, let's change those tests, to avoid unnecessary code paths in actual non-test code